src/responsive-breakpoints/responsive-breakpoints.js

```diff
@@ -0,0 +1,22 @@
+window.addEventListener("resize", setBodyCssClasses);
```

Comment:
Don't you need to import this file in the entry file?

Reply:
Ah yep, good catch. It’s working in storybook because I imported it into the file, but I forgot to test it in the app. Will change